datafusion/expr/src/expr_fn.rs

```diff
@@ -501,7 +501,7 @@ scalar_expr!(
 scalar_expr!(Degrees, degrees, num, "converts radians to degrees");
 scalar_expr!(Radians, radians, num, "converts degrees to radians");
 nary_scalar_expr!(Round, round, "round to nearest integer");
-scalar_expr!(Trunc, trunc, num, "truncate toward zero");
+nary_scalar_expr!(Trunc, trunc, "truncate toward to precision");
```

Comment:
I don't think this function truncates toward `precision` -- instead it truncates towards zero with an optional precision

```suggestion
nary_scalar_expr!(Trunc, trunc, "truncate toward zero, with optional precision");
```